tox.ini

```diff
@@ -37,6 +37,22 @@ deps =
 commands =
     python -m paasta_tools.run-paasta-api-in-dev-mode
 
+[testenv:playground-api]
+envdir = .tox/py37-linux/
+passenv = PAASTA_TEST_CLUSTER KUBECONFIG PAASTA_SYSTEM_CONFIG_DIR PAASTA_API_SOA_DIR
+setenv =
+    KUBECONFIG = ./k8s_itests/kubeconfig
+    PAASTA_TEST_CLUSTER = kind-emanelsabban-k8s-test
```

Comment:
Maybe we should rename the test cluster to something more generic before merging this?